qa/tasks/ses_qa.py

```diff
@@ -221,10 +222,16 @@ def ses_ceph_osd_tiering(self, **kwargs):
                 )
 
     def ses_disk_fault_injection(self, **kwargs):
+        _remote = get_remote_for_role(self.ctx, 'osd.0')
```

Comment:
I don't follow by what convention do you  name local variables with underscore prefix?